src/labels/CustomLabels.labels

```diff
@@ -2421,7 +2421,7 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Label that appears when user does not pass FLS check to copy the address</shortDescription>
-        <value>You don&apos;t have the required field permissions. Contact your system administrator to inquire about getting the necessary permissions.</value>
+        <value>You don&apos;t have the required object or field permissions. Contact your system administrator to inquire about getting the necessary permissions.</value>
```

Comment:
@sam-knox , can you please advise on the short description if it is required to change?